packages/bundle-source/README.md

```diff
@@ -57,6 +57,19 @@ with `@preserve`, `@copyright`, `@license` pragmas or the Internet Explorer
 Comment elision does not strip comments entirely.
 The syntax to begin or end comments remains.
 
+## TypeScript type erasure
+
+TypeScript modules with the `.ts`, `.mts`, and `.cts` extensions in
+packages that are not under a `node_modules` directory are automatically
+converted to JavaScript through type erasure using
+[`ts-blank-space`](https://bloomberg.github.io/ts-blank-space/).
+
+This will not function for packages that are published as their original
+TypeScript sources, as is consistent with `node
+--experimental-transform-types`.
+This will also not function properly for TypeScript modules that generate
+JavaScript runtime code, like enums.
```

Comment:
"enums" aren't necessarily TypeScript. (JSDoc has `@enum`). 
```suggestion
This will also not function properly for TypeScript modules that have [runtime impacting syntax](https://github.com/bloomberg/ts-blank-space/blob/main/docs/unsupported_syntax.md), such as `enum`.
```